libr/util/p_format.c

```diff
@@ -1388,10 +1388,19 @@ int r_print_format_struct_size(const char *f, RPrint *p, int mode, int n) {
 			break;
 		case '?':
 			{
+			const char *wordAtIndex = "";
```

Comment:
I believe the default value should be NULL, otherwise in case you enter the `if (idx < words)` wordAtIndex would still point to "" and the `if (!wordAtIndex) { break;}` is not executed.